project/Dependencies.scala

```diff
@@ -839,14 +839,16 @@ object Dependencies {
   val http4sCirce = "org.http4s" %% "http4s-circe" % http4sV
   val pact4sScalaTest = "io.github.jbwheatley" %% "pact4s-scalatest" % pact4sV % Test
   val pact4sCirce = "io.github.jbwheatley" %% "pact4s-circe" % pact4sV
+  val pact4sSpray = "io.github.jbwheatley" %% "pact4s-spray-json" % pact4sV
 
   val pact4sDependencies = Seq(
     pact4sScalaTest,
     pact4sCirce,
+    pact4sSpray,
```

Comment:
I don't love that we now have two JSON serialization libraries here, but I didn't want to change either all the DRS contract tests (which use Circe) or the WorkflowCallbackActor (which uses Spray).